modules/diskq/logqueue-disk-non-reliable.c

```diff
@@ -224,6 +224,7 @@ _ack_backlog(LogQueue *s, gint num_msg_to_ack)
         return;
       msg = g_queue_pop_head(self->qbacklog);
       POINTER_TO_LOG_PATH_OPTIONS(g_queue_pop_head(self->qbacklog), &path_options);
+      log_queue_memory_usage_sub(&self->super.super, log_msg_get_size(msg));
```

Comment:
When rewinding, we move messages from qbacklog to qout.
My understanding is that we should not do anything with memory_usage counters as qout_pop will decrease that for us.

Reply:
I agree that if we are moving between memory queues, we should not modify the value of the memory usage counter.
But here, we are in `ack_backlog()`, not in `rewind_backlog()`, where messages are removed from the `qbacklog`, and not pushed to another one, so I think we should decrease the counter.